examples/example.py

```diff
@@ -11,9 +11,7 @@ def example():
     # Note: Use with for the example testing, doesn't need to be like this on the README
```

Comment:
Suggestion : `example` is a bit too generic a method name. All this does is run one episode. I would rename the method to `run_one_episode` and make the config an argument (which defaults to `configs/tasks/rearrange/pick.yaml` if none is given).

Reply:
(probably out of scope for this PR now that read more of it)